packages/@react-native-windows/codegen/src/generators/AliasGen.ts

```diff
@@ -71,7 +71,7 @@ interface AliasCodeMap {
 
 function getArrayTypeName(
   type: Nullable<NativeModuleBaseTypeAnnotation>): string {
-    if (type.type === 'ArrayTypeAnnotation' && type.elementType?.type === 'TypeAliasTypeAnnotation') {
```

Comment:
was this throwing an error?

Reply:
Meta changed their type declarations which adjusted the value of type.elementType in this scenario to always have a value and not be null. So the ? in type.elementType can be removed (left in it causes a linting error because the check if its null is no longer necessary).